content/swift/concepts/dictionaries/dictionaries.md

```diff
@@ -113,3 +77,7 @@ for meaning in emojiMeaning.values {
   print(meaning)
 }
 ```
+
+Below are some instance properties for dictionaries:
+
+<!--   TODO: Add methods, and subscripts  -->
```

Comment:
@Dusch4593 what do you think? We can make a bunch of issues here, requesting for different swift methods and subscripts. @lucwhite do you agree that this may be a good initiative for hacktoberfest?

Reply:
I +1 this! 👍🏻